fl4health/strategies/basic_fedavg.py

```diff
@@ -132,6 +132,8 @@ def configure_fit(
             if self.on_fit_config_fn is not None:
                 # Custom fit config function provided
                 config = self.on_fit_config_fn(server_round)
+            else:
```

Comment:
In a few examples, we don't specify a config, but we assume that `current_server_round` is always present. So this (and those below) ensure that it is in there.